addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -65,11 +72,63 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
         });
     },
 
+    /**
+     * Check if the confirmation form is valid by testing each of its inputs
+     *
+     * @private
+     * @param $form
+     * @return {boolean} - true if no errors else false
+     */
+    _isConfirmationFormValid($form) {
+        const formErrors = [];
+
+        $form.find('.form-group').each(function (index, field) {
+            let $field = $(field);
+            let inputs = $field.find('.form-control');
+            let invalidInputs = inputs.toArray().filter(function (input) {
+                return !input.checkValidity();
+            });
+
+            inputs.removeClass('is-invalid');
+            if (invalidInputs.length) {
+                inputs.addClass('is-invalid');
```

Comment:
Strange that you are adding "is-invalid" on ALL inputs instead of just the invalid ones, or did I miss something?

Reply:
The `inputs` are the ones inside a `form-group` not all the ones inside the `<form>` but indeed it's not necessary here since there is only one input for each `form-group`